Jack/Property.fs

```diff
@@ -133,33 +203,47 @@ module Property =
         | None ->
             printfn "+++ OK, passed %s." (renderTests tests)
             true
-        | Some (nshrinks, msgs) ->
+        | Some (report, nshrinks) ->
             printfn "*** Failed! Falsifiable (after %s%s%s):"
                 (renderTests tests)
                 (renderShrinks nshrinks)
                 (renderDiscards discards)
-            List.map (printfn "%s") msgs |> ignore
+            List.map (printfn "%s") (Report.toList report) |> ignore
             false
 
-    let check (p : Property) : bool =
+    let check (p : Property<unit>) : bool =
         check' 100 p
 
 [<AutoOpen>]
-module ForAllBuilder =
+module PropertyBuilder =
```

Comment:
Yes! I find `Property` a better name, since we may not only support universal quantification in the future. 😉 

One day, if we decide to support existentials (à la [SmallCheck](https://github.com/feuerbach/smallcheck)), we may still do that through some custom operation(s), and then we won't have to rename the builder itself – which may be a breaking change if we follow [SemVer](http://semver.org/).